x/delayedack/keeper/invariants.go

```diff
@@ -1,64 +1,71 @@
 package keeper
 
 import (
+	"errors"
 	"fmt"
 
+	errorsmod "cosmossdk.io/errors"
 	sdk "github.com/cosmos/cosmos-sdk/types"
-
+	"github.com/dymensionxyz/dymension/v3/utils/uinv"
 	commontypes "github.com/dymensionxyz/dymension/v3/x/common/types"
 	"github.com/dymensionxyz/dymension/v3/x/delayedack/types"
 	rtypes "github.com/dymensionxyz/dymension/v3/x/rollapp/types"
 )
 
-const (
-	routeFinalizedPacket = "rollapp-finalized-packet"
-)
+var invs = uinv.NamedFuncsList[Keeper]{
+	{Name: "proof-height", Func: InvariantProofHeight},
+}
 
-// RegisterInvariants registers the delayedack module invariants
-func (k Keeper) RegisterInvariants(ir sdk.InvariantRegistry) {
-	// INVARIANTS DISABLED SINCE LAZY FINALIZATION FEATURE
+func RegisterInvariants(ir sdk.InvariantRegistry, k Keeper) {
+	invs.RegisterInvariants(types.ModuleName, ir, k)
 }
 
-// PacketsFinalizationCorrespondsToFinalizationHeight checks that all rollapp packets stored are set to
-// finalized status for all heights up to the latest height.
-func PacketsFinalizationCorrespondsToFinalizationHeight(k Keeper) sdk.Invariant {
-	return func(ctx sdk.Context) (string, bool) {
-		var (
-			broken bool
-			msg    string
-		)
+// DO NOT DELETE
```

Comment:
> // DO NOT DELETE

why u need this warning?

Reply:
someone like me might delete it because it appears unused
But SDK says you need to define it
I will try to dig out the doc why